### DISCUSSION TITLE
Camilla DSP 2.0

### DISCUSSION THREAD

**Discussion by ddogrollthereal** [rern/rAudio#1564] [Fixed Issues]
I just updated to the latest raudio and it seems to work however I cannot configure it.
Just get error "Errors: Expected ',' or ']' after array element in JSON at position 240 (line 11 column 28)" every time I go to the DSP page.
It is enabled and passing audio.

I've also performed Pacman -Syyu via SSH and rebooted issue has remained unchanged and I'm so far unable to locate the offending json file, I will keep looking though.

Any help would be appreciated.
![Screenshot 2024-01-11 124944](https://github.com/rern/rAudio/assets/156258654/e923fe62-acc4-46ba-9fa4-d309410d0c42)

**Comment by rern** [comment]
Please try:
```sh
sed -i "s#system/camilladsp#& | tr ' ' ,#" /srv/http/bas
h/settings/camilla-data.sh
```

**Comment by ddogrollthereal** [comment]
That worked well.

Thank you
